### ISSUE TITLE
Setting format: "markdown" yields typescript error

### ISSUE THREAD

**Issue by moorage** [scrapfly/typescript-scrapfly#9]
Have this code in my typescript file, but getting a `Type '"markdown"' is not assignable to type 'Format | undefined'`:

```ts
import { ScrapflyClient, ScrapeConfig, ScrapeResult, log } from "scrapfly-sdk";

    const scrapeConfig = new ScrapeConfig({
      url,
      asp: true,
      country: "US", 
      render_js: true, 
      format: "markdown", // ERROR: Type '"markdown"' is not assignable to type 'Format | undefined'
    });
    ```

**Comment by mazen-r** [comment]
Hey @moorage, the ScrapeConfig `format` accepts a `ScrapeFormat` enum:
```typescript
import { ScrapflyClient, ScrapeConfig, ScrapeResult, log, ScrapeFormat } from "scrapfly-sdk";

    const scrapeConfig = new ScrapeConfig({
      url,
      asp: true,
      country: "US", 
      render_js: true, 
      format: ScrapeFormat.MARKDOWN
    });
    ```
```
A new release will be deployed soon to allow strings directly.

**Comment by Granitosaurus** [comment]
released v0.6.7 which allows to use strings for these Enum options. Thanks for opening the issue @moorage 🙏
